src/main/scala/zio/rocksdb/transaction/ZTransaction.scala

```diff
@@ -0,0 +1,35 @@
+package zio.rocksdb.transaction
+import org.rocksdb.WriteOptions
+import org.{ rocksdb => jrocks }
+import zio._
+
+final class ZTransaction private (jTransaction: jrocks.Transaction) extends RocksDB.TransactionService {
```

Comment:
Let’s name this `Transaction`. We haven’t prefixed anything else in the library with `Z`.

Reply:
Will conflict with https://github.com/zio/zio-rocksdb/pull/13/files#diff-dda252e0cfc4debfb3205f637def10acR8

Reply:
Renamed to `LiveTransaction`